### ISSUE TITLE
:crop="false" bug

### ISSUE THREAD

**Issue by rafuck** [alessiomaffeis/vue-picture-input#38]
If crop=false sometimes image scales incorrectly. My fix at drawImage in the !crop branch:
```
        if (this.imageRatio > 1) {
          scaledHeight = scaledWidth / this.imageRatio
          if (scaledHeight > this.previewHeight){
            scaledHeight = this.previewHeight;
            scaledWidth  = scaledHeight * this.imageRatio;

            offsetX = (this.previewWidth - scaledWidth) / 2
          }
          else{
            offsetY = (this.previewHeight - scaledHeight) / 2
          }
        } else {
          scaledWidth = scaledHeight * this.imageRatio
          if (scaledHeight > this.previewHeight){
            scaledWidth  = this.previewWidth;
            scaledHeight = scaledWidth / this.imageRatio;

            offsetY = (this.previewHeight - scaledHeight) / 2
          }
          else{
            offsetX = (this.previewWidth - scaledWidth) / 2
          }
        }
```

**Comment by alessiomaffeis** [comment]
Hi! First of all, thanks for contributing. Can you please provide an example of the issue? It would be awesome if you could attach some screenshots too! :)

**Comment by rafuck** [comment]
Ok. Lets use this settings:
```
<picture-input width="800" height="240" :crop="false"  ...  ></picture-input>
```
And for example this image (446x335)
![baraban](https://user-images.githubusercontent.com/2648737/31501618-69a03622-af73-11e7-97de-888344cbbd18.png)
`imageRatio = 446/335 = 1.33 > 1` and on resize we obtain:
```
// Initial values
scaledWidth = previewWidth = 800;
scaledHeight = previewHeight = 240;
// Calculated
scaledHeight = scaledWidth / this.imageRatio = 800/1.33 = 601.5;
```
So, our "scaled" height is greater than preview containers height.

**Comment by rafuck** [comment]
Also I have a few comments not related to this issue, but I dont know if it's convenient to bring them here

**Comment by alessiomaffeis** [comment]
Thanks for the example, I will try to replicate the issue. :)
Are your comments related to the project? If that's the case, you can easily create a separate issue, otherwise, you can send me an email.

**Comment by alessiomaffeis** [comment]
It should be fixed in v1.18.1, try it out! Let me know if it doesn't work, and I'll reopen the issue.
